neptune/new/__init__.py

```diff
@@ -32,7 +32,7 @@
     NeptuneUninitializedException,
     NeptunePossibleLegacyUsageException,
 )
-from neptune.new.run import Run
+from neptune.new.metadata_containers import Run, Project, Model, ModelVersion
```

Comment:
Haven't we come to a conclusion NOT to do this?

Reply:
Yes we did, I'l fix.